cogboard-webapp/src/components/UserLogin.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { TextField, Button, IconButton, Typography } from '@material-ui/core';
+import { AccountCircle, PowerSettingsNew } from '@material-ui/icons';
+import { useDialogToggle, useFormData } from '../hooks';
```

Comment:
Move thunks and hooks imports above the component imports. Generally we have 3 types of imports in the following order separated by line breaks: libs, own helpers, hooks etc. and components (3rd party components and own).